### ISSUE TITLE
When is SFMesh support coming for VS2019??

### ISSUE THREAD

**Issue by MickBadran** [Azure/service-fabric-mesh-preview#307]
Just upgraded to VS.NET 2019 Preview 1 - trying to get SFMesh to insall....I've got a better chance of seeing Santa.

Just wondering when.

**Comment by MikkelHegn** [comment]
@BigMorty

**Comment by BigMorty** [comment]
@MickBadran - A VS2019 Mesh tooling extension is on our backlog!

**Comment by aloneguid** [comment]
You can always manually install the extension, it's a matter of changing manifest file.

**Comment by BigMorty** [comment]
I don't think that will work for VS2019, I believe there are some code changes we need to make in order for the Mesh extension to work.

**Comment by BigMorty** [comment]
@MickBadran - I believe @aloneguid was referring to changing the manifest file in the vsix to identify it supports VS2019. Unfortunately, there are other changes we need to make to the Mesh extension to work properly in VS2019.

**Comment by aloneguid** [comment]
Yeah sorry I've tried to do that, but unfortunately VSIX depends on some packages available only in VS2015 so it crashes on install. We'll wait for your updates @BigMorty ;)

**Comment by BigMorty** [comment]
👍👍👍

**Comment by Farzad-Jalali** [comment]
> You can always manually install the extension, it's a matter of changing manifest file.

Could you please kindly show us how to do it?

**Comment by sondreb** [comment]
Mesh Tooling for Visual Studio 2019 has been on the backlog for almost a year now, any updated?

**Comment by kassemdarwish** [comment]
Any updates on this?
